### PR TITLE
Fixing typos

### PR DESCRIPTION
**Pull request by deining**
This PR fixes a few typos I spotted on the site.

### CODE REVIEW

**Review by masinter — APPROVED** [review]
changing the typos in the bibliography won't help -- those are downloaded (by a currently manual process) from our Zotero bibliography. I'll submit a bibliography issue.

I'm imagining we'll want to move the stories from "medley goals? to the "What People are saying" discussion. But even so, fixing typos is good.

**Review by masinter — APPROVED** [review]
hopefully the commit will leave enough breadcrumbs to find the bibliography typos.

File: data/bibliography.json [hunk 1]
```diff
@@ -2506,7 +2506,7 @@
...
+      "abstract": "We describe enhancements to graphical search and replace that allow users to extend the capabilities of a graphical editor. Interactive constraint-based search and replace can search for objects that obey user-specified sets of constraints and automatically apply other constraints to modify these objects. We show how an interactive tool that employs this technique makes it possible for users to define sets of constraints graphically that modify existing illustrations or control the creation of new illustrations. The interface uses the same visual language as the editor and allows users to understand and create powerful rules without conventional programming. Rules can be saved and retrieved for use alone or in combination. Examples, generated with a working implementation, demonstrate applications to drawing beautification and transformation.",
```
Comment (nbriggs) [inline data/bibliography.json:2509]: Also an error in the text published in the ACM DL.

File: data/bibliography.json [hunk 2]
```diff
@@ -1387,7 +1387,7 @@
       "container-title": "ACM SIGART Bulletin",
       "page": "31–32",
       "issue": "77",
-      "abstract": "The Interslip-D project was formed to develop a personal machine inplementation of Interlisp for use as an environment for research in artificial intelligence and cognitive science [Burton et al., 80b]. This note describes the principal developments since our last report almost a year ago [Burton et al., 80a].",
+      "abstract": "The Interslip-D project was formed to develop a personal machine implementation of Interlisp for use as an environment for research in artificial intelligence and cognitive science [Burton et al., 80b]. This note describes the principal developments since our last report almost a year ago [Burton et al., 80a].",
```
Comment (nbriggs) [inline data/bibliography.json:1390]: Interlisp-D, but this and the "inplementation" are in the document on the ACM DL.  I have not found a scanned image of the Bulletin, so I don't know whether these are transcription errors or actually present in the original text.

**Review by stumbo — APPROVED** [review]
Updated Zotero Bibliography fixing errors noted in bibliography.json  (Interlisp/medley#1156)

**Pull request merged**